### ISSUE TITLE
Make the popping of custom notification ad more noticeable to user

### ISSUE THREAD

**Issue by aseren** [brave/brave-browser#33603]
System notifications on MacOS slides from the right edge. We can possibly do the same.

**Comment by tmancey** [comment]
@aguscruiz your thoughts on this suggestion? Thanks

**Comment by aguscruiz** [comment]
Yes, this is a good idea. Also if this applicable to Window's notifications (whatever animation they have) we should do it too
